### PR TITLE
App crashed when i select and crop image for profile. #691

### LINKED ISSUE #691: App crashed when i select and crop image for profile.
Describe the bug.
When I tried to select an image for my profile I clicked on the upload and the complete action. I selected the image I want to use and click on the crop function immediately I clicked on crop the app crashed.

### Screen recording.
https://youtu.be/cwIxaIMT6_8

### Devices and Versions)

* Device: [Gionee, X1]

* Version [7.0]

* App version [5.5.0]

### PR DESCRIPTION
**Pull request by sukhbeer**
## Description
fix the issue with minor modification.

Fixes #691 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue Jul 03 11:13:27 CEST 2018
+#Sun Aug 18 12:32:39 IST 2019
```
Comment (Swati4star) [inline Android/gradle/wrapper/gradle-wrapper.properties]: revert this change please

File: Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java [hunk 2]
```diff
@@ -232,6 +232,8 @@ protected void onCreate(Bundle savedInstanceState) {
         });
         displayStatus.addTextChangedListener(mCountCharacters);
         changeImage.setOnClickListener(v -> {
+            Intent getIntent = new Intent(Intent.ACTION_GET_CONTENT);
+            getIntent.setType("image/*");
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java]: how is this change fixing the issue? You are not even using `getIntent` anywhere

File: Android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (sukhbeer) [reply Android/gradle/wrapper/gradle-wrapper.properties]: okay

File: Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java (hunk 2 above)
Comment (sukhbeer) [reply Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java]: Sorry I'll fix this shortly
